package/gluon-respondd/src/respondd-nodeinfo.c

```diff
@@ -101,6 +101,9 @@ struct json_object * respondd_provider_nodeinfo(void) {
 	struct json_object *software_firmware = json_object_new_object();
 	json_object_object_add(software_firmware, "base", gluon_version());
 	json_object_object_add(software_firmware, "release", gluonutil_wrap_and_free_string(gluonutil_read_line("/lib/gluon/release")));
+	json_object_object_add(software_firmware, "target", json_object_new_string(platforminfo_get_target()));
```

Comment:
I think these should use `gluonutil_wrap_string` - some of these platforminfo functions can return NULL.

Reply:
ups, my mistake. Didn't apply that from your earlier suggestion